sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2964,6 +2972,17 @@ public Write<T> withPrimaryKey(List<String> primaryKey) {
       return toBuilder().setPrimaryKey(primaryKey).build();
     }
 
+      /** Specify how missing values should be interpreted when there is a default value in the schema. Options are to
+       * take the default value or to write an explicit null (not an option of the field is also required.). Note: this
+       * is only used when using one of the storage write API insert methods.
```

Comment:
Can we add some validation that fails when this feature is set with other write methods?